skxray/speckle_visibility/speckle_visibility.py

```diff
@@ -0,0 +1,290 @@
+# ######################################################################
```

Comment:
@tacaswell I am developing analysis softwares for XSVS(X-ray speckle visibility spectroscopy) and it will use some of the functions here and other functions before doing XSVS analysis. Therefore I thought it is better to have it separate XSVS analysis?